.github/workflows/release.yml

```diff
@@ -48,13 +48,14 @@ jobs:
       run: ./gradlew --stacktrace --no-configuration-cache publishAndReleaseToMavenCentral
 
     - name: Generate Release Notes
-      # Temporary implementation for the very first release. The release notes for this will be written by hand as they
-      # would otherwise contain the complete commit history.
-      run: echo "Initial release!" > RELEASE_NOTES.md
-      #run: ./gradlew -q printChangeLog > RELEASE_NOTES.md
+      run: ./gradlew -q printChangeLog > RELEASE_NOTES.md
```

Comment:
This seems to print the commits in groups, right? We could consider something more sophisticated like https://keepachangelog.com/en/1.0.0/ as well. I'd say especially as we are quite strict with the commits, using all commits for changelog is not that nice.

Reply:
> This seems to print the commits in groups, right?

Yes, it's the same mechanism as at https://github.com/oss-review-toolkit/ort/releases.

Reply:
I believe that proper user-facing release notes have to be handwritten, but for now I'm happy to at least have a fully automated solution.